src/language/index.md

```diff
@@ -1,90 +1,463 @@
 ---
-title: Introduction to the Dart language
+title: Introduction to Dart
 description: A brief introduction to Dart programs and important concepts.
 short-title: Dart basics
 ---
 
-These pages show you how to use each major Dart feature, from
-variables and operators to classes and libraries, with the assumption
-that you already know how to program in another language.
-For a briefer, less complete introduction to the language, see the
-[language samples page](/samples).
+This page provides a brief introduction to the Dart language
+through samples. 
 
-To learn more about Dart's core libraries, see the
-[library tour](/guides/libraries/library-tour).
-Whenever you want more details about a language feature,
-consult the [Dart language specification][].
+For more details on the Dart language, see the individual language
+pages in the menu.
 
-## A basic Dart program
+You might also want to check out the 
+[library tour](/guides/libraries/library-tour)
+or the [Dart cheatsheet codelab](/codelabs/dart-cheatsheet).
 
-The following code uses many of Dart’s most basic features:
 
-<?code-excerpt "misc/test/language_tour/basic_test.dart"?>
+## Hello World
+
+Every app has a `main()` function.
+To display text on the console, you can use the top-level `print()` function:
+
+<?code-excerpt "misc/test/samples_test.dart (hello-world)"?>
 ```dart
-// Define a function.
-void printInteger(int aNumber) {
-  print('The number is $aNumber.'); // Print to console.
+void main() {
+  print('Hello, World!');
 }
+```
 
```

Comment:
Might want to be more clear what "arguments" are here, to avoid confusion with function-call arguments. I believe you mean the program or command-line arguments:

```suggestion
Read more about [the `main()` function][] in Dart,
including optional parameters for program arguments.

[the `main()` function]: /language/functions#the-main-function
```

```suggestion
Read more about [the `main()` function][] in Dart,
including optional parameters for command-line arguments.

[the `main()` function]: /language/functions#the-main-function
```

Reply:
Yes, that's better! I just quickly skimmed the content in that section to grab something to list after "including..."; there wasn't much